CONTRIBUTING.md

```diff
@@ -1,6 +1,32 @@
-# Web World Wind Design and Coding Guidelines
+# Contributing to Web WorldWind
 
-## General
+The following are a set of guidelines for contributing to the NASA Web WorldWind project. These are mostly guidelines, not
```

Comment:
My original thought was led by the idea of "open-source", in which anyone in the community could suggest changes. However, given the nature of our project, I see how we might not need these opening sentences--especially since these are guidelines that the team adheres to. I am not set on keeping them.